.github/ISSUE_TEMPLATE/launch-scope.md

```diff
@@ -0,0 +1,6 @@
+Description
+Define the tasks that will lead a successful launch of this project, create tickets for those tasks, and address any known unknowns pertaining those tasks.
+
+Proposed Solution
```

Comment:
Yeah, it might be good to add a task that’s along the lines of:
>- [ ] In one or two sentences, describe what “launch” consists of for this project. Some examples:
>    - “This project will be launched in a brand new production environment within the client’s own Platform.sh account. Once the project has been QAed and approved by the client, the `my-cool-product.com` domain’s DNS records will need to be updated by the client’s IT staff.”
>    - “Launching this project will consist of enabling new features X, Y, and Z in the client’s existing Acquia production environment after said features have been QAed and approved by the client in a staging environment. After enabling these features, migrations A, B, and C will need to be manually triggered and monitored by Chromatic.”
>    - “The launch process will begin by granting access to this repo to the client’s internal devops staff, and sending an email message to the team with instructions on how to build all of the necessary assets. The client’s devops staff is responsible for the eventual deployment, which they will schedule ahead of time. Chromatic will be available for support.”

I don’t know if that’s too much, but I want to convey: 1) that “launch” can mean wildly different things and that is OK, so long as we clearly define it, 2) that this short description is intended to give anyone reading it a very quick overview of what it’ll take to launch the project in question.

What I’m not sure of is: where does that definition of “launched” go? 🤔

Reply:
I like where your head is at. Thanks for the suggestions. Let me know what you think of my revisions.